src/Redux/features/AddTodoSlice.jsx

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = [
+  { id: 1, message: "great", checked: false },
+  { id: 2, message: "Read Redux", checked: false },
+  { id: 3, message: "Read Anki", checked: false },
+];
```

Comment:
I suggest that you can at least create a single initial value

Reply:
- Why?